getting-started/master-your-ui/the-table-view.md

```diff
@@ -36,6 +36,59 @@ The **extended search** lets you search also within _reference fields_ of `belon
 _Extended search_ is **not the default** search because it is **slower**.
 {% endhint %}
 
+### Advanced syntax for search
+
+{% hint style="info" %}
+_Advanced syntax for search_ is only supported on nodejs agents `@forestadmin/agent` starting from version 1.37. 
+{% endhint %}
+
+Search supports advanced syntax to help you find exactly what you are looking for:
+
+* `-term` will search for records that do not contain `term` ;
+* `property:term` will search for records that contain `term` in the `property` field ;
+* `relation.childProperty:term` will search for records that contain `term` in the `childProperty` field of the `relation` relation ;
+* `term1 OR term2` will search for records that contain `term1` or `term2` ;
```

Comment:
Does OR, AND, NULL and such have to be uppercase ?

Reply:
Yes. I added the precision in the docs. Thanks